src/Client/Client.cs

```diff
@@ -0,0 +1,123 @@
+using Newtonsoft.Json;
+using System;
+using System.Buffers;
+using System.IO.Pipelines;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TcpEcho
+{
+    public class BlockchainTransactionGet {
+        [JsonProperty("id")]
+        public int Id { get; set; }
+        [JsonProperty("result")]
+        public string Result { get; set; }
+    }
+
+    public class Client
```

Comment:
nitpick: let's rename this class from Client to TcpClient

Reply:
actually, disregard this because there seems to be another class called TcpClient already